modules/storage-persistent-storage-pv.adoc

```diff
@@ -249,6 +249,8 @@ cluster.
 
 |===
 
+To find the time of the last phase transition, look at the parameter, `LastPhaseTransitionTime`, which has a timestamp that is updated every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`).
```

Comment:
I think this reads better as two sentences, maybe something like this:

```adoc
Use the `LastPhaseTransitionTime` parameter to find the time of the last phase transition. This parameter contains a timestamp that updates every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`).
```

This also removes the passive voice ("is updated").

Reply:
^^ This sounds good, but do we use the term "parameter" in this context when talking about object specs? I think it should be "field" or "nested field".

Reply:
+1 for 'Field' instead of  'parameter'